k6_test/grpc/grpc_client.js

```diff
@@ -0,0 +1,45 @@
+import { check } from "k6";
+import { Counter } from "k6/metrics";
+import grpc from "k6/net/grpc";
+
+const grpcReqs = new Counter("grpc_reqs");
```

Comment:
@aluu317 do you know why this is needed? I thought k6 already counts the requests itself?

Reply:
but to answer your question, yes, k6 logs it, something like this:
```
 ✗ status is OK
      ↳  0% — ✓ 0 / ✗ 272

     █ setup

       ✗ status is OK
        ↳  0% — ✓ 0 / ✗ 1

     █ teardown

     checks...............: 0.00%  ✓ 0          ✗ 273 
```
So 273 is total checks, but unsure how many for each VU